src/lhttpc_manager.erl

```diff
@@ -271,7 +271,7 @@ init(Options) ->
         true ->
             % Make sure that the ssl random number generator is seeded
             % This was new in R13 (ssl-3.10.1 in R13B vs. ssl-3.10.0 in R12B-5)
-            apply(ssl, seed, [crypto:rand_bytes(255)]);
+            apply(ssl, seed, [crypto:strong_rand_bytes(255)]);
```

Comment:
Not only is it undocumented, [it's not actually exported or defined](https://github.com/erlang/otp/blob/OTP-19.3.4/lib/ssl/src/ssl.erl) as of OTP 17, 18 and 19. Perhaps we can just throw the seeding call away? I'm guessing it was part of `ssl` at some point, but removed since.